src/Microsoft.NetCore.Analyzers/UnitTests/Data/ReviewSQLQueriesForSecurityVulnerabilitiesTests.cs

```diff
@@ -1580,20 +1586,20 @@ End Sub
         }
 
         [Fact]
-        public void MissingWellKnownTypes_NoDiagnostic()
+        public async Task MissingWellKnownTypes_NoDiagnostic()
```

Comment:
💭 I'm not sure this test is still correct. You should make sure the test fails if assembly is provided, by adding a using alias directive that references a type defined in System.Data (ideally one of the types used by the analyzer).